.well-known/apple-app-site-association

```diff
@@ -0,0 +1,15 @@
+{
+  "applinks": {
+    "details": [
+      {
+        "appIDs": ["chat.delta", "chat.delta.desktop.electron"],
+        "components": [
+          {
+            "#": "*&i*",
```

Comment:
this is to ensure it is an invite link to still allow you to visit the generator page